lib/cherry_pick.js

```diff
@@ -120,68 +100,6 @@ export default class CheckPick {
     }
   }
 
-  async downloadAndPatch(expectedCommitShas) {
-    const { cli, repo, owner, prid } = this;
-
-    cli.startSpinner(`Downloading patch for ${prid}`);
-    // fetch via ssh to handle private repo
-    await runAsync('git', [
-      'fetch', `git@github.com:${owner}/${repo}.git`,
```

Comment:
In `LandingSession`, there is `https://...` instead. The comment here seems to indicate the SSH fetch was intentional.